lib/pusher-platform/authentication_response.rb

```diff
@@ -0,0 +1,29 @@
+module PusherPlatform
+  class AuthenticationResponse
+    attr_reader :status, :headers, :body
+
+    def initialize(options)
+      @status = options[:status]
+      @headers = options[:headers] || {}
+      @body = options[:body]
+    end
+
+    def to_s
+      "PusherPlatform::AuthenticationResponse - status: #{@status} body: #{@body.to_json}"
+    end
+
+    def as_json(options = {})
+      json = {
+        status: @status,
+        headers: @headers,
+        body: @body
+      }
+      json
```

Comment:
Nitpicking – not sure why you can't just return the object without assignment.

Reply:
Good spot - think it was split from what I was doing some logging but forgot to kill it. Will do now 👍